src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -330,6 +330,17 @@ RoverPositionControl::control_position(const matrix::Vector2d &current_position,
 	return setpoint;
 }
 
+float RoverPositionControl::lowpass_filter(const float sample_rate, const float cutoff_req, float input)
```

Comment:
There are ready made lowpass filters in px4, e.g. a second order one in mathlib: src/lib/mathlib/math/filter/LowPassFilter2p.hpp

Why not use one of those?

Reply:
sure, I am gonna switch to one of the ready made one when this is confirmed a solution. I was just experimenting with this LPF idea if it would help with the stuttering motion.

Reply:
LPF is actually not a solution

Reply:
LPF is actually not a solution